core/src/components/router/router.tsx

```diff
@@ -61,7 +61,17 @@ export class Router implements ComponentInterface {
     await waitUntilNavNode();
     console.debug('[ion-router] found nav');
```

Comment:
The comment is off-topic to the issue but I noticed that the router has some console traces that could not be de-activated.

Is it intended ?

Reply:
My guess is that they were added during the development of Ionic Framework v4 3+ years ago and never removed.  While they should not show up in production builds, we probably don't need them in development builds either.

Reply:
Are you going to fix it in the PR or do you want me to submit an other PR specifically for that ?

Reply:
I can remove it in this.